webpack.config.js

```diff
@@ -37,7 +37,7 @@ module.exports = (env, argv) => {
         // Don't run minification for CI builds (this is only set for runs on develop)
         // We override this via environment variable to avoid duplicating the scripts
         // in `package.json` just for a different mode.
-        argv.mode = "development";
+        nodeEnv = "development";
```

Comment:
I probably would revert that change, looking at the webpack documentation for [`mode`](https://webpack.js.org/configuration/mode/) it will set the `NODE_ENV` variable to what you expect + do a few useful thing for us

```suggestion
        argv.mode = "development";
```